packages/webapp/src/main/services/diagram/diagram-repository.ts

```diff
@@ -52,4 +52,22 @@ export const DiagramRepository = {
       }
     });
   },
+  convertSvgToPdf(svg: string, width: number, height: number): Promise<Blob | null | undefined> {
```

Comment:
Can you try to omit `null` since there's already `undefined`?

Reply:
Removed null.